Lesson7/deniz/index.html

```diff
@@ -1,11 +1,211 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">
+    <title>Medium2</title>
 </head>
 <body>
+    <header>
+
+    </header>
+
+    <main>
+        
+        
+        <div class="container-top">
+
+          <div class=content-one>  
+                <h1 class="h1-top">Time-saving CSS techniques to create responsive images</h1>
+            
+                <div class="profile-info">
+                       <div class="profil-photo">
+                            <img class="small-pic" src="./img/IMG_7036.png" alt="Adrien Zaganelli"> 
+                            </div>
+            
+                            <div class="top-div">
+                            <div class="Adrien-link">
+                            <a href="#">Adrien Zaganelli</a>
+                            <a href="#"><p class="fbtn">Follow</p></a>
+                        </div> 
+                            <div class="date-min">
+                                <span>Published in</span> <a href="#">We've moved to FreecodeCamp.org/news</a><span>3 min read</span><span>Aug 12,2018</span>
+                            </div>  
+               </div>
+            </div>     
+           
+        </div>         
+            
+       
+            <div class="icons">
+                <div class="div-sol">
+                    <a href="#"><i class="fa-solid fa-hands-clapping"><p>10.4K</p></i></a>
+                    <a href="#"><i class="fa-regular fa-comment"><p>21</p></i></a>
+                
+            </div>
+                  <div>
+                    <a href="#"><i class="fa-regular fa-bookmark"></i></a>
+                    <a href="#"><i class="fa-regular fa-circle-play"></i></a>
+                    <a href="#"><i class="fa-solid fa-arrow-up-from-bracket"></i></a>
+                  </div>
+            </div>
+
+
+            <div class="image-container">
+                  <div class="clock-pic">
+                        <img class="clock-img" src="./img/1_aVzJTznRRfP1lM7AXe9yLw.webp" width="1000" height="751" alt="A man who works with a laptop ona clock">
+                        <figcaption>Photo by <a href="#">Kevin Ku </a><a href="#">on Unsplash</a></figcaption>
+                  </div> 
+            </div>
+        
+         
+            <div class="p-container">
+                <div class="p-inside">
+                   <p>As a web developer, there is a high probability that you have encountered the two enemies of this article: images and deadlines. Sometimes, for some reasons, your images won’t fit the layout and you don’t want to wrap your head around this for hours.</p>
+                   <p>This situation has happened to me many times and I have learned from my mistakes. No more black magic hacks — here are my five favorites techniques to handle image resizing.</p>
+                   <h1 class="">the “OMG I NEED THIS ASAP” way</h1>
+                   <pre>
```

Comment:
There's an HTML code element, use it instead to display computer code: https://www.w3schools.com/html/html_computercode_elements.asp